Content/LeagueSandbox-Scripts/Buffs/Global/Recall.cs

```diff
@@ -55,16 +58,20 @@ public void OnUpdateMoveOrder(IObjAiBase unit)
             var buff = unit.GetBuffWithName("Recall");
             if (buff != null)
             {
-                buff.DeactivateBuff();
+                if (unit.MoveOrder != OrderType.Hold && unit.MoveOrder != OrderType.Stop)
```

Comment:
Is it that `unit.MoveOrder` will never miss the current move order? (say for example that I update the move order very fast with many spells/taunt and so on).

Reply:
There are race conditions in the case of multiple Listeners, where one of them changes the MoveOrder too. This will cause all the listeners that are called after to miss this update event and rely on a new OrderType. We need to choose what kind of behavior we expect.

Reply:
I don't see what you mean. All listeners are called per-update.

Reply:
@microsoftv Let's assume that there are 2 listeners: A,B. A changes the move order in some way (for example, calls StopMovement). That will change unit.MoveOrder. Now the code for B starts and it uses different unit.MoveOrder.

Reply:
Alright, then ideally the event should have the order when triggered. I would prefer if we PR'd this one alongside your PR, as the recall in this PR removes the listener.

Reply:
My PR will never get merged (or at least not in the near term future) as there are no active reviewers besides me. I suggest that you will just take my code from there and copy it in your PR. If you want you can add "coauthored by" line to the PR and I'll add it to the commit.

Reply:
Done.